### PR TITLE
WIP: Pool Number Refresh

### PR DESCRIPTION
**Pull request by sreynen**
This will be a script to refresh a number pool, removing numbers in no-longer-used area codes, and also replacing numbers that have newer alternatives in the same area code. The script will be optional, as not everyone using Spoke will have a need to rotate numbers, and can be set to whatever schedule is appropriate via Cron, e.g. orgs might want to rotate more often during higher usage periods to mitigate increased risk of numbers getting blacklisted.

### CODE REVIEW

File: src/workers/pool-number-refresh.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
+import Twilio from 'twilio'
+// import { r } from '../server/models'
+
+/**
+ * Get all numbers contacted in last 30 days.
+ */
+const getRecentContactNumbers = () => {
+  // TODO: replace this with database call after thinky to knex switch
```
Comment (shakalee14) [inline src/workers/pool-number-refresh.js:8]: could we also do a call to the twilio api?

File: src/workers/pool-number-refresh.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+}, {})
+
+/**
+ * Get numbers for Twilio poool from API.
+ */
+const getCurrentTwilioPoolNumbers = (twilio, sid) => {
+  const service = twilio.messaging
+    .services(sid)
+    .fetch()
+  return service
```
Comment (shakalee14) [inline src/workers/pool-number-refresh.js]: when we return service, does that return an object that has available phone numbers in it?

Comment (sreynen) [reply src/workers/pool-number-refresh.js]: Maybe? This part doesn't currently work at all. It's what https://www.twilio.com/docs/api/rest/messaging-services says should work, but in Spoke code, `twilio.messaging` is undefined.

File: src/workers/pool-number-refresh.js (hunk 1 above)
Comment (sreynen) [reply src/workers/pool-number-refresh.js:8]: Probably, but since we have the numbers in the database, that should be faster and more reliable.

**Comment by sreynen** [comment]
Turns out the TODOs on using the Twilio API can't been done with the current API, so this is stalled until new API features arrive.

**Pull request closed without merging**